policy-service/src/main/java/uk/gov/gchq/palisade/service/policy/request/Policy.java

```diff
@@ -38,7 +38,9 @@
  * service but not needed by the rest of the palisade services. That includes
  * separating the rules that need to be applied at the resource level or the record level.
  *
- * @param <T> the java class to deserialise the record into, used for typing rules (UnaryOperators on T)
+ * @param <T> The Java class that the rules expect the records of data to be in
+ *            the format of, e.g. a Policy Int has Rules Resource for coarse filtering
+ *            and Rules Int for fine filtering of files containing Ints
```

Comment:
Do you think this reads better?
```suggestion
 * @param <T> The Java class that the rules expect the records of data to be in
 *            the format of, e.g. if T was String then a policy has Rules<Resource> for coarse filtering
 *            and Rules<String> for fine grain filtering of files where each record is a String
```

Reply:
Yeah, but checkstyle will treat <...> as an open tag and fail with no closing </...> tag